test/e2e/federated-ingress.go

```diff
@@ -58,8 +61,56 @@ const (
-	FederatedIngressTLSKey = `LS0tLS1CRUdJTiBFTkNSWVBURUQgUFJJVkFURSBLRVktLS0tLQpNSUlGRGpCQUJna3Foa2lHOXcwQkJRMHdNekFiQmdrcWhraUc5dzBCQlF3d0RnUUlxQjFIRXpocDREOENBZ2dBCk1CUUdDQ3FHU0liM0RRTUhCQWlKREszNkNoOUpmd1NDQk1naXM1UkJ1ZTZXekU0elN5TlVNQ25yeFJsOW9GbHoKWHlLNXJXMkc0R1VTeG8rcFhvYWxoSEd6YVlHaElsUXZZWW42U0ZSTmcrRWpFanBxM2MyY25iZURLN1lvakkrTgpkdW1neEludkJoSUZlM1d6VVBCUFo1bVZ5L1FkRzRpUlQ3Rm9FZHIwUjVKcGdHMnd2M0hSblhQUFNoOTc1Y0lZCm5QcTBiSGRmT2R4ZVJnYWliVHZyeWJ5WGxiZFRFdEhSWWpLYnFnL1RvWkRuQXRib04yaWJLa1NkQityN2lFcWEKV3Q5VnpEalNuM0hhOER2UW0wNVQ2VThuYWtiTTFnbTlwUnNyMThveE4yTktKcVZPdnMvY1JSYzFEWTQxWDBjZwpoTEh6Um9QNytvY21PMlQ4dE5mKzdRMXEwUTBpNUM4enpHWndtdFNkc2kvcHluN2NkUmJIeDdqNXErTy9nNWkyClBRSUN1cHRtY04rZjJvOVBBSzdkUXpnY2VNK3BTUUdpQlBXWXhvSEtXM0ZhbnVVWlcxMS9jVjVDMGUwTGFzcFYKa2NaeFZKdlpqQ0E3QUdaN0dIOXB2UVNDRkdjQml6MDI4eTE0bFkwWEQ0VjZENDZabk40L3lvdmJNZEsrZ1l6WApmY201aVUwRlhBMytCd08rK2tJczBuNkpScFlMTlA1Z2NTRXhpelhXVDkyYTVyQzlVaXJGRmMrQzRiSXI1L0czCnFmK3ZsVmp0bWZLTDBJd29GY2hLWWFaTHlrNnZ2WTlMOE43T21KQVhoVXE2dWFnODV3Y1cvRTc3NlpZd1J4MGgKMFRSNHhDcFo2WThDdi9EdWZlUEZnWUc0eXkxckdkd2ZNOVJBQzFVNkliZFczVnFEcWNRdVZBcjRzeVhMc0FESAppUXFaQjNvRStzQzZKRTYvek1EWGxoNXpOZWhpNjNuZUhORmYxQjE1YjdDcWJmL1ZVUUdnaHdkSnI1NWNiK0pFCjBmQUtXcFhtYmVYcEVlamhmdmpwU2tuS1RPajU5aHlqVnVuaGxxb2NxMm0zR1hXc3lOUTFiRGtIVFhjcGVhSGkKNU5TNFZ4TnJjVDc5UGRXQlkrUjhPUXFBN1FXZURnaWNtd0VnbkpPdHVlN25pYzd6b1VtKzlTZ3hSUC9HZ2dUVAplaGkrR2wyNzgrUUJnV2E4eTFPWTdraVNJSjBPZFl6KzlzRjYyTzBlMWVuSFBacHYycFF0S0VtSGlaNjA1OHFpCmhXQ2RZb2hPeEdmMXJOSEhlSWM5bGJDOUwxbTFSY3F0aU9JZ01zR3FhT2VpV2k5VFR2aVVlV2ZLK0FVUEs4eDEKeUNscUpiK1U2T1hMZE9lNHM2MlcvQ1pJcTViWGhrYzFwTndQT2NJTFBoMG13U3pwWDljdDBEQ01TTzVsRGQwWApJcUVxL3o1WnBvajJNQjMwaUJIak9YRGY3N1EwaGtCVDRhZ3V2Rjl0VEFWMWwyd1djZU5CVEQvcVdqWm54TDQ4CkxGWlhMbkxISC9UY0tYRjV0ZVdlbjJzY3dXYWF2bmJPQXNLYzV4QzVMU0NHU3hXd2I2alVmeGlmSGJJbzlWdzcKeTVILzR2L2ZlOHB1VExtSVFQZWx6c1BrcnVQQTJSNVMySTl3WDFPb1BoK3FPVndoUzliRk5ZUm5xYWt0WDBWOQpka1FMSG9JOENNT2Z0Nkxqam1Uc1N4NFB2NXpGR2srUkdQOFdHR25KbDdXemdZbDBYYjdydVRTclkzcE1TUFQxCmJSOUsxcFpmQ0J1ajdrckdUTGc2b2xIVWU4N215QTRKaXNHSzI5bVExbkV4ZGJnbCtpVithUjVJQjIzTkFxOC8KUHA0TjJyUlRlYUE4bEM1dTk2bnBiblF2ZEdYMXJickFQeDRpTk13bE94K3Vaa1lrVDFRZmRUekpINUtBNkRsRApNUWJjNnFlVjVPRjNyNUQ2VzBiLzZ1TUFWekE2dURkZm53b3ZFRkQzYUU5SnRnOTA4MW1Xa2RmVXI3L081aXIxClplSUVvUktvcngzYzhvKytvTFpEZ09kemo2WUFSUld4UDZweDlTeHpTM0NHK09XcnBwZWxkREVxV3ZkVlVEd3EKTHdHMDJ2Rms0VHhWVG5MWVhLVkl6R0oraHlGcjgxbXR3ZXBsanFzcEJZSDFhQUQyVk12bWhaUktXYzEzY0xCbQowaTA9Ci0tLS0tRU5EIEVOQ1JZUFRFRCBQUklWQVRFIEtFWS0tLS0tCg==`
+	FederatedIngressTLSCrt = `-----BEGIN CERTIFICATE-----
```

Comment:
Needs file reading and the associated error handling. Plus you need to ensure that the right files get copied to the test tarballs. It is just too painful. I would just leave it here. Also, it is a commonly used test pattern. Ex: https://github.com/golang/go/blob/master/src/crypto/tls/tls_test.go#L26